### ISSUE TITLE
bug: The rendering of the elements hidden behind the keyboard is delayed while scrolling and then appears when scrolling stops.

### ISSUE THREAD

**Issue by maravedimike** [ionic-team/ionic-framework#19430]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**
If I have a row of images (or even a div with a gradient) that is offscreen/behind the keyboard when I scroll down (or up) to it, they do not render properly when the keyboard minimizes. The elements do not appear until I lift my finger off the screen or when the scroller finishes all its animations.

This is causing a super noticeable problem for me, making the whole thing look choppy. Is there any way in which I can prevent this choppy-ness from happening?

[Here is a gif of the issue.][1]

  [1]: https://gfycat.com/sneakyuniquebighorn

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/39659118/65452195-53496600-de06-11e9-8545-092416de8e03.gif)

**Expected behavior:**
The expected behavior is for the elements behind the keyboard to quickly render and appear on the screen when the keyboard minimizes. 
```

**Comment by nicdichiara** [comment]
I am having this issue. 

In the Template:
`<ion-content
  [scrollEvents]="true"
  (ionScrollStart)="closeKeyboard();"
>`

In the .TS
` closeKeyboard() {
    try {
      Keyboard.hide();
    } catch (error) {}
  }`

It goes black like in the GIF above and seems like it's actually closing on scrollend? Am I implementing the desired action incorrectly or could it be an issue with the scroll event firing timing with the keyboard?

**Comment by nicdichiara** [comment]
We were able to work around this by using touchstart instead.
`<ion-content (touchstart)="closeKeyboard()">`
with
`closeKeyboard() {Keyboard.hide();}`

Although, this created an issue with inner ion-content click events so we ended up modifying it to trigger the keyboard close by losing focus on the ion-searchbar. (It would be useful to have exposed a setBlur() event like the existing setFocus()) We implemented a "setBlur()" by creating a function: 
`loseFocus() {
    const activeElement = <HTMLElement>document.activeElement;
    if (activeElement && activeElement.blur) {
      activeElement.blur();
    }
  }` 
inspired by this solution: https://forum.ionicframework.com/t/dismiss-keyboard-on-touch-outside-it/66998/4?u=grabitadmin

We then just set the (touchstart)="loseFocus()" and placed it in a closer encapsulating div to reduce any conflicts. If you implement this, I would recommend directly addressing the input or searchbar instead of doing it globally like this -- it just happened to be the case that this doesn't cause any conflicts in our situation.
